### PR TITLE
Update adaptiveCardPrompt.ts

### PR DESCRIPTION
**Pull request by v-kydela**
## Description

Mostly formatting changes and refactoring.

### CODE REVIEW

**Comment by mdrichardson** [comment]
Cool cool. I haven't reviewed it yet, but:

* Have you run the tests for it?
* Do you plan to make the same changes to C#?
* If so, would you prefer I review this one first, or both together once C# is done?

File: libraries/botbuilder-dialogs/src/prompts/adaptiveCardPrompt.ts [hunk 1]
```diff
@@ -168,7 +168,7 @@ export class AdaptiveCardPrompt extends Dialog {
 
     public async beginDialog(dc: DialogContext, options: PromptOptions): Promise<DialogTurnResult> {
         // Initialize prompt state
-        const state: any = dc.activeDialog.state as PromptState;
+        const state: any = dc.activeDialog.state as AdaptiveCardPromptState;
```
Comment (mdrichardson) [inline libraries/botbuilder-dialogs/src/prompts/adaptiveCardPrompt.ts]: This is my fault, but can you change `any` to `AdaptiveCardPromptState`, then remove `as AdaptiveCardPromptState`

File: libraries/botbuilder-dialogs/src/prompts/adaptiveCardPrompt.ts [hunk 2]
```diff
@@ -206,7 +206,7 @@ export class AdaptiveCardPrompt extends Dialog {
     // Override continueDialog so that we can catch activity.value (which is ignored, by default)
     public async continueDialog(dc: DialogContext): Promise<DialogTurnResult> {
         // Perform base recognition
-        const state: PromptState = dc.activeDialog.state as PromptState;
+        const state: AdaptiveCardPromptState = dc.activeDialog.state as AdaptiveCardPromptState;
```
Comment (mdrichardson) [inline libraries/botbuilder-dialogs/src/prompts/adaptiveCardPrompt.ts]: Also my fault. Remove `as AdaptiveCardPromptState`. I believe it's redundant.

**Review by mdrichardson — CHANGES_REQUESTED** [review]
Just a couple pedantic changes that were my fault anyway. Overall, great code review. Nice cleanup.

**Review by mdrichardson — APPROVED** [review]
Nice! Tests pass

**Comment by mdrichardson** [comment]
@v-kydela 	Were you planning on making the same updates to the C# version? I can; just don't want to double-commit

**Comment by v-kydela** [comment]
@mdrichardson - You go ahead

**Pull request merged**